db/schema.rb

```diff
@@ -311,6 +311,7 @@
     t.datetime "status_check_completed_at", comment: "The last time a status check was successfully completed"
     t.boolean "ready_for_status_check", default: false
     t.datetime "notification_sent_at", comment: "The time a notification was sent"
+    t.datetime "last_batch_claim_at"
```

Comment:
Well, I suppose this is due to how I envisioned the action, since the enrollment was picked up at a specific time, by a specific batch_claim, as opposed to the individual record _being_ batch_claimed. I'll go along with the past tense though to keep it consistent.